app/src/main/res/values/styles.xml

```diff
@@ -403,7 +403,14 @@
         <item name="android:textColor">?primaryText</item>
     </style>
 
+    <style name="Body12TextStyle" parent="TextAppearance.MaterialComponents.Body1">
+        <item name="android:textColor">@color/photonDarkGrey05</item>
```

Comment:
It's not the `primaryText` color.. I'll have to see if one exists for `photonDarkGrey05` and if not I'll set one up

Reply:
looks like "Jump back in" tabs is using a pre-defined color as well (`<color name="home_show_all_button_text">@color/photonDarkGrey05</color>`). I'm going to keep this as-is for now and fix them together in a separate PR when we get the Dark values

Reply:
I got the dark theme colors from Nicole. Fixed it here and [filed a new issue](https://jira.mozilla.com/browse/FNX-22747) to add them to "Jump back in"

Reply:
Ah yes, we just added `home_show_all_button_text`. It is using `photoLightGrey50` for the dark theme in the "Show all" button - see https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/values-night/colors.xml#99 (note the `values-nightly/colors.xml`). I checked with Nicole to about header colours since I didn't see this respond before. I still hold that we probably want to keep the `textColor` as `?primaryText` in these `BodyXXTextStyle` since it would be strange to have "12" be different from the rest and we can just add the `textColor` locally.

Reply:
The "Jump back in" header actually also uses the correct dark theme color with using `style="@style/Header20TextStyle"`. In our existing color variables and component system, `?primaryText` maps to https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/values-night/colors.xml#7 which maps to https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/values/colors.xml#80 which is `photonLightGrey05`. I will close the Jira issue since there's no change necessary.

You might be interested in reusing `home_show_all_button_text` since we anticipated the need for a generic "Show all" button text color.